src/main/kotlin/com/hypto/iam/server/service/PasscodeService.kt

```diff
@@ -157,9 +157,12 @@ class PasscodeServiceImpl : KoinComponent, PasscodeService {
         try {
             sesClient.sendTemplatedEmail(emailRequest)
         } catch (e: SesException) {
-            if (e.statusCode() == HttpStatusCode.BAD_REQUEST) {
-                throw BadRequestException("Bad request values")
+            val exceptionMessage: String? = e.message
+            if (exceptionMessage != null && exceptionMessage.contains("Domain contains illegal character")) {
+                logger.error("Email contains illegal characters")
+                throw BadRequestException("Domain contains illegal character")
```

Comment:
Can we say something like `Invalid email address: $email_address`
The present message might confuse our users, like what is domain